src/middleware.js

```diff
@@ -5,17 +5,6 @@ import { get, set } from 'lodash';
 import omitBy from 'lodash/omitBy';
 
 
-// where morgan connects to winston
-function addStream(logger, level) {
```

Comment:
I never understood the meaning of this function before but after reading the morgan code I do. morgan isn't designed to work with a full-fledged logger, but a stream (basically an object with an interface that exposes a `write` function). We were passing morgan a logger which we were wrapping as a stream (exposing a `write` function) on a logger ojbect that itself probably had a stream somewhere under the hood. We were adding a few layers of direction and it probably wasn't necessary. Simply removing it will remove `level` from the logs morgan generates so I will go back and fix that.